tests/framework/pkg/session/session.go

```diff
@@ -9,9 +9,10 @@ type CleanupFunc func() error
 
 // Session is used to track resources created by tests by having a LIFO queue the keeps track of the delete functions.
 type Session struct {
-	cleanupQueue []CleanupFunc
-	open         bool
-	testingT     *testing.T
+	CleanupEnabled bool
```

Comment:
The constructor should default this to true.

Reply:
fixed